commit d96809e9910d7b735d5cdc79cdc87f8495a5a67d
Author: Ivan Trepakov <liontiger23@gmail.com>
Date:   Mon Nov 3 12:04:07 2025 +0700

    Workaround https://savannah.gnu.org/bugs/?63852
    
    Disable tests that fail under root due to reliance on
    write permission manipulation which does not work with root

diff --git a/tests/scripts/features/output-sync b/tests/scripts/features/output-sync
index 3353f1c6..f69821b2 100644
--- a/tests/scripts/features/output-sync
+++ b/tests/scripts/features/output-sync
@@ -380,17 +380,17 @@ unlink($fout);
 # Create a non-writable temporary directory.
 # Run the test twice, because run_make_test cannot match a regex against a
 # multiline input.
-my $tdir = 'test_tmp_dir';
-mkdir($tdir, 0500);
-$ENV{'TMPDIR'} = $tdir;
-
-run_make_test(q!
-all:; $(info hello, world)
-!, '-Orecurse', "/suppressing output-sync/");
-
-run_make_test(undef, '-Orecurse', "/#MAKE#: 'all' is up to date./");
-
-rmdir($tdir);
+#my $tdir = 'test_tmp_dir';
+#mkdir($tdir, 0500);
+#$ENV{'TMPDIR'} = $tdir;
+#
+#run_make_test(q!
+#all:; $(info hello, world)
+#!, '-Orecurse', "/suppressing output-sync/");
+#
+#run_make_test(undef, '-Orecurse', "/#MAKE#: 'all' is up to date./");
+#
+#rmdir($tdir);
 }
 
 # This tells the test driver that the perl test script executed properly.
diff --git a/tests/scripts/features/temp_stdin b/tests/scripts/features/temp_stdin
index fee32a90..84a357a2 100644
--- a/tests/scripts/features/temp_stdin
+++ b/tests/scripts/features/temp_stdin
@@ -115,16 +115,16 @@ rmdir($tmakedir);
 # makefile from stdin to a temporary file.
 # Create a non-writable temporary directory.
 
-my $tdir = 'test_tmp_dir';
-mkdir($tdir, 0500);
-$ENV{'TMPDIR'} = $tdir;
-close(STDIN);
-open(STDIN, "<", 'input.mk') || die "$0: cannot open input.mk for reading: $!";
-
-run_make_test(q!
-all:; $(info hello, world)
-!, '-f-', '/cannot store makefile from stdin to a temporary file.  Stop./', 512);
-rmdir($tdir);
+#my $tdir = 'test_tmp_dir';
+#mkdir($tdir, 0500);
+#$ENV{'TMPDIR'} = $tdir;
+#close(STDIN);
+#open(STDIN, "<", 'input.mk') || die "$0: cannot open input.mk for reading: $!";
+#
+#run_make_test(q!
+#all:; $(info hello, world)
+#!, '-f-', '/cannot store makefile from stdin to a temporary file.  Stop./', 512);
+#rmdir($tdir);
 }
 
 # This close MUST come at the end of the test!!
